server/src/routes/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,12 @@
+class ReviewSerializer {
+  static async getSummary(review) {
+    const allowedAttributes = ["id", "comment", "rating", "trailId", "userId"]
+
+    let serializedReview = {}
+    for (const attribute of allowedAttributes) {
+      serializedReview[attribute] = review[attribute]
+    }
```

Comment:
If we wanted the User who made the Review, we could query for that here (note that you would need a UserSerializer!)